src/app/services/data.service.js

```diff
@@ -135,12 +135,15 @@
             var deferred = $q.defer();
 
             connect().then(function () {
-                servConn.getHistory(item, {
-                    instance: service,
+                servConn.getHistory(item, { //it seems that this function always goes to history.0 or to default history instance defined in system
```

Comment:
do we need to fix this in backend? so other history instances could also be used?

Reply:
as bluefox mentioned, the getHistory service is using the configured one in admin/system of IOB. To have it more generic it would be a good idea to have it extended in the backend. It would be also an option to use the sendTo instead getHistory and then the flexibility is also given. But sendTo is first time part of this PR.